src/select/utils/render-options.tsx

```diff
@@ -65,11 +65,14 @@ export const renderOptions = ({
     const ListItem = useInteractiveGroups ? MultiselectItem : Item;
     const isSticky = firstOptionSticky && globalIndex === 0;
 
+    // Adjust virtual position to create 1px overlap between consecutive selected items in multiselect
+    const adjustedVirtualPosition = virtualItem ? virtualItem.start - index : undefined;
```

Comment:
Should we have a multiselect test page, too?

Potentially, we can create some top-level "virtual-scroll" test page that can feature both select and multiselect, or even additionally include autosuggest and property filter.